mixins/elasticsearch.yml

```
@@ -28,7 +28,12 @@ actions:
         export PAPI_ENV_ID="${globals.papiEnvId}"
         export PAPI_API_VERSION="${globals.papiApiVersion}"
 
-        sleep 60 # Required in case the cron get_metadata_from_HN hasn't run yet
+        # Required in case the cron get_metadata_from_HN hasn't run yet
+        while [ ! -f /metadata_from_HOST ]
+        do
+          sleep 1
+        done
```

to be a little more assured of having the data (with our usual formatting):
```suggestion
        while [ ! -s /metadata_from_HOST ]; do
          sleep 1
        done
```

---

:heavy_check_mark: